AbletonMCP_Remote_Script/handlers/devices.py

```
@@ -0,0 +1,1360 @@
+        if ctrl:
+            ctrl.log_message("Error deleting device: " + str(e))
+        raise
+
+
+# --- Macro helpers (new from MacWhite) ---
+
+
+def get_macro_values(song, track_index, device_index, track_type="track", ctrl=None):
+    """Get the values of all macro controls on a rack device."""
+    try:
+        track, device = _resolve_device(song, track_index, device_index, track_type)
```

Ruff (0.15.0)

[warning] Unpacked variable `track` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)